### PR TITLE
[MODULAR] Bring back the aesthetics PDA in GAGS form,also aesthetics PDA cartridge

### PR DESCRIPTION
**Pull request by Sonic121x**
## About The Pull Request
image
some design are diffrent than the orignal aesthetics PDA sprite.
and the captain PDA have its own sprite now.because i just can't make it look right.
it wasn't as nice as it used to be.if anyone have any idea how to make them look better please let me know.

## Why It's Good For The Game
those sprite are just look nicer in my own opinion,and these are used to be the PDA sprite anyway.

## Changelog
:cl:
add: aesthetics PDA in GAGS
add: aesthetics cartridge
/:cl:

### CODE REVIEW

**Comment by GoldenAlpharex** [comment]
~~Hey, while you're at it, do you think you could do it for the cartridges as well? I absolutely loved the new sprites we had for those, and the /tg/ ones are absolutely horrendous, let's face it.~~
Never mind I'm an idiot that can't read. Godspeed, this is amazing.

**Comment by GoldenAlpharex** [comment]
On a sidenote, did you make sure that the Curator one didn't have a bugged sprite when you insert the ID inside of it? I know it was an issue with the last iteration of these sprites.

File: modular_skyrat/modules/aesthetics/pda/pda.dm [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#dab424#262d99#65c9ff"
+
+/obj/item/pda/heads/cmo
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#d6d9e0#262d99#65c9ff"
+
+/obj/item/pda/heads/rd
+	greyscale_config = /datum/greyscale_config/pda/aesthetics/stripe_thick/head
+	greyscale_colors = "#d6d9e#262d99#c28be5"
```
Comment (GoldenAlpharex) [inline modular_skyrat/modules/aesthetics/pda/pda.dm]:
```suggestion
	greyscale_colors = "#d6d9e0#262d99#c28be5"
```
You forgot the 0 here and I suspect that's why you're failing integration tests.

**Comment by Sonic121x** [comment]
> On a sidenote, did you make sure that the Curator one didn't have a bugged sprite when you insert the ID inside of it? I know it was an issue with the last iteration of these sprites.

yea,i set the overlay offset to 0 and it should fix the problem
```
/obj/item/pda/curator
	overlays_x_offset = 0
```
image

**Review by Gandalf2k15 — APPROVED** [review]

**Comment by GoldenAlpharex** [comment]
Hey @Sonic121x, would you mind hitting me up on one of the development Discord channels of the server? There's a bug with the PDAs and I'd like to discuss it with you ;)

**Pull request merged**